core/controllers/topic_editor_test.py

```diff
@@ -273,9 +303,15 @@ def test_editable_topic_handler_put(self):
             }, {
                 'change_affects_subtopic_page': True,
                 'cmd': 'update_subtopic_page_property',
-                'property_name': 'html_data',
-                'old_value': '',
-                'new_value': '<p>New Value</p>',
+                'property_name': 'page_contents_html',
```

Comment:
I didn't find any test for handling changes through another command `page_contents_audio`? can we add a test to see changes made through another command is getting properly handled?

Reply:
done